donate/templates/pages/core/campaign_page.html

```diff
@@ -75,4 +75,19 @@ <h1 class="heading heading--primary">
             </div>
         </div>
     </div>
+
+    {% if recaptcha_site_key %}
+    <div id="g-recaptcha" data-public-key="{{ recaptcha_site_key }}"></div>
+    {% endif %}
+
+{% endblock %}
+
+{% block extra_js %}
+    {{ currencies|json_script:"currencies" }}
+    {{ braintree_params|json_script:"payments__braintree-params" }}
+    <script src="https://www.paypalobjects.com/api/checkout.min.js" data-version-4></script>
+    <script src="{% static '_js/payments-paypal.compiled.js' %}"></script>
+    {% if recaptcha_site_key %}
+    <script src="https://www.google.com/recaptcha/api.js?render=explicit" async defer></script>
+    {% endif %}
```

Comment:
We don't need to re-define the extra_js block, we should only need to inclued the g-recaptcha div in the campaign page content block.

Reply:
(This template inherits from landing_page.html, and so already contains the scripts)

Reply:
aaah, right! Updated